packages/cli/src/commands/doctor/doctor.ts

```diff
@@ -1,18 +1,19 @@
-// @flow
 import chalk from 'chalk';
+// @ts-ignore
```

Comment:
Noobie question: shouldn't we type these instead of just adding it to ignore?

Reply:
Yep we should. I might give it a try, but there is a pending WIP PR on the repo. Would probably be worth being put on DefinitelyTyped